sdk/data/azcosmos/cosmos_query_response.go

```diff
@@ -130,3 +130,14 @@ func newDatabasesQueryResponse(resp *http.Response) (QueryDatabasesResponse, err
 type queryDatabasesServiceResponse struct {
 	Databases []DatabaseProperties `json:"Databases,omitempty"`
 }
+
+// ReadManyItemsResponse contains response from the item query operation.
+type ReadManyItemsResponse struct {
+	Response
```

Comment:
I'm hesitant to return a `Response` as part of this, because it sets the expectation that this API maps to a single HTTP request/response pair, even though it almost never will. For example, what headers do we include on the `Response` here?

There's a similar issue, IMO, with query responses today in the Go SDK, but since that's already released as public API, we can't do much about it.

My thinking is we just expose the items and the total RU charge for now, and add other data as needed later.